libs/messenger-bundle/tests/TestEnvVarsTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\MessengerBundle\Tests;
+
+trait TestEnvVarsTrait
```

Comment:
Tohle by si uz zaslouzilo nekam presunout :) Asi bych na to zalozil task.

Reply:
Jo, tuhle jsem s Odinem dosli ke stejnemu zaveru. Zalozil jsem https://keboola.atlassian.net/browse/PST-1058